.github/workflows/test.yml

```diff
@@ -0,0 +1,183 @@
+          - php: '5.4'
+            wp: '3.7'
+            experimental: false
+          - php: '5.3'
+            wp: '5.1'
+            experimental: false
+          - php: '5.3'
+            wp: '3.7'
+            experimental: false
+
+          # Complement the builds run via the matrix with some additional builds against specific WP versions.
+          - php: '7.4'
+            wp: '5.4'
+            experimental: false
+          - php: '7.3'
+            wp: '5.3'
+            experimental: false
+          - php: '7.2'
+            wp: '5.2'
+            experimental: false
+          - php: '7.1'
+            wp: '4.6'
+            experimental: false
+          - php: '7.0'
+            wp: '4.3'
+            experimental: false
+          - php: '5.6'
+            wp: '4.0'
+            experimental: false
+          - php: '5.6'
+            wp: '5.5'
+            experimental: false
+          - php: '5.5'
+            wp: '4.5'
+            experimental: false
+          - php: '5.4'
+            wp: '5.0'
+            experimental: false
+          - php: '5.3'
+            wp: '4.9'
+            experimental: false
+
+          # Experimental builds. These are allowed to fail.
+          - php: '7.4'
+            wp: 'trunk'
+            experimental: true
+
+          - php: '8.1'
+            wp: 'trunk'
+            experimental: true
+
+    name: "Test: PHP ${{ matrix.php }} - WP ${{ matrix.wp }}"
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    services:
+      mysql:
+        # WP 4.5 is the first WP version which properly supports MySQL 5.7.
+        # See: https://core.trac.wordpress.org/ticket/34692
+        # WP 5.4 is the first WP version which largely supports MySQL 8.0.
+        # See: https://core.trac.wordpress.org/ticket/49344
+        #image: mysql:${{ ( matrix.wp < 4.5 && '5.6' ) || ( matrix.wp < 5.4 && '5.7' ) || '8.0' }}
+        image: mysql:5.6.50
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: false
+          MYSQL_DATABASE: wordpress_test
+        ports:
+          - 3306/tcp
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          # The PHP 5.6 and 7.0 images don't include mysql[i] by default.
+          extensions: mysqli
+
+      - name: Start mysql service
+        run: sudo /etc/init.d/mysql start
+
+      - name: Verify mysql version
+        run: |
+          mysqld --version
+          mysqladmin -V
+
+      - name: Set up WordPress
+        run: phpunit/install.sh wordpress_test root root 127.0.0.1 ${{ matrix.wp }}
+
+      - name: Determine supported PHPUnit version
+        id: set_phpunit
+        run: |
+          # Originally, only PHPUnit 4.x and 5.x were supported.
```

Comment:
If the comment is included as a part of the `run` command, it will be printed out to the [first line of the step](https://github.com/WordPress/wordpress-importer/runs/1615689255?check_suite_focus=true#step:8:1).

Not a huge deal, but this docblock could be moved above the name to make the step's output a bit cleaner.